daemon/cmd/daemon.go

```diff
@@ -994,15 +1003,6 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 		d.bootstrapClusterMesh(nodeMngr)
 	}
 
-	// Must be done at least after initializing BPF LB-related maps
-	// (lbmap.Init()).
```

Comment:
Shouldn't this comment be preserved in addition to the new one you added? This context is useful to have if it's still relevant.